### PR TITLE
Use fast mirror for playground installation

### LINKED ISSUE #16: Change ubuntu apt-get source list for Chinese developers
As mentioned in #13 , we should change the ubuntu apt-get install source for Chinese developers to provision vagrant environment quickly.

### PR DESCRIPTION
**Pull request by lhw362950217**
Fixes #16 #17

### CODE REVIEW

File: provision.bash [hunk 1]
```diff
@@ -2,24 +2,37 @@
 
 set -e  # Exit script if any error
 
+# The shared folder is specified in Vagrantfile.
+VAGRANT_SHARED_FOLDER=/home/vagrant/desktop
+
+. $VAGRANT_SHARED_FOLDER/sqlflow/docker/dev/find_fastest_resources.sh
```
Comment (Yancey0623) [inline provision.bash:8]: Can remove the prefix `.`.

File: start.bash [hunk 2]
```diff
@@ -14,6 +14,11 @@
 
 set -e
 
+echo "Docker pull SQLFlow images ..."
```
Comment (Yancey0623) [inline start.bash:17]: > Docker pull SQLFlow images ...

Pull the SQLFlow Docker image sqlflow/sqlflow ...

**Review by Yancey0623 — APPROVED** [review]
LGTM with tiny comment.

File: provision.bash (hunk 1 above)
Comment (lhw362950217) [reply provision.bash:8]: here, we use . to import the function from `find_fastest_resources.sh` according to https://stackoverflow.com/questions/9772036/pass-all-variables-from-one-shell-script-to-another

**Pull request merged**